crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -720,6 +720,56 @@ impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
         self
     }
 
+    /// Tries to add a [`Bundle`] of components to the entity.
```

Comment:
There should be a line in the `insert` docs linking to `try_insert` as a non-panicking alternative.